middleware/logging.go

```diff
@@ -83,6 +91,9 @@ func (l Log) Wrap(next http.Handler) http.Handler {
 		statusCode, writeErr := wrapped.getStatusCode(), wrapped.getWriteError()
 
 		if writeErr != nil {
+			if errors.Is(writeErr, DoNotLogError{}) {
```

Comment:
This isn't right: the `writeError` is an error from `Write()`, not the underlying operation error.